functorch/csrc/DynamicLayer.cpp

```diff
@@ -254,8 +244,8 @@ static Tensor unwrapIfDead(const Tensor& tensor) {
   return wrapped->value();
 }
 
-void foreachTensorInplace(std::vector<IValue>& args, int64_t begin, int64_t end,
-    std::function<Tensor(const Tensor&)> func) {
+void foreachTensorInplaceWithIdx(std::vector<IValue>& args, int64_t begin, int64_t end,
```

Comment:
nit: quick comment that this calls the function with increasing idx would be good, I had to confirm by reading the code

Reply:
Also, is this used anywhere?

Reply:
Bleh nope, I needed this previously but ended up not needing it after refactoring and forgot to remove it. I'll remove it (and make `foreachTensorInplace` do exactly what it was doing before)